pilot/pkg/model/gateway.go

```diff
@@ -210,3 +212,23 @@ func IsPassThroughServer(server *networking.Server) bool {
 
 	return false
 }
+
+// convert ./host to currentNamespace/Host
```

Comment:
What happened with ~ ?

Reply:
~ is for the global export/import flags.. in another PR